### PR TITLE
feat(Client): create Client

### PR DESCRIPTION
**Pull request by KurokuTetsuya**

### CODE REVIEW

**Comment by KurokuTetsuya** [comment]
`build` function not implemented until we have http server (express)

**Review by Waxaranai — COMMENTED** [review]
Please fix some the eslint fail

File: src/structures/Client.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { createLogger } from "../utils/Logger";
+import * as config from "../config";
+
+export class Client {
+    public readonly config = config;
+    public readonly logger = createLogger();
+    // eslint-disable-next-line
+    constructor() {}
```
Comment (Hazmi35) [inline src/structures/Client.ts:8]:
```suggestion
    // eslint-disable-next-line @typescript-eslint/no-empty-function, @typescript-eslint/no-useless-constructor
    public constructor() {}
```
Only disable specific rules

**Review by Hazmi35 — CHANGES_REQUESTED** [review]
Please see above

**Review by Hazmi35 — APPROVED** [review]
LGTM

**Review by Allvaa — APPROVED** [review]

**Review by Waxaranai — APPROVED** [review]

**Pull request merged**